### DISCUSSION TITLE
List has where filter function?

### DISCUSSION THREAD

**Discussion by xietuhuo** [typst/typst#3877] [Q&A] [answered]
I want to customized my special list (like marker with sym.triangle.filled),but when I use show **list.where(marker:blabla) :set rule** ,it has no effect. can't List have the more detailed  management.

**Comment by EpicEricEE** [comment, marked as answer]
The filter is a bit weird, as you have to wrap the marker in a content block. This works:
```typ
#show list.where(marker: [#sym.triangle.filled]): set text(red)

#list(marker: sym.triangle.filled,
  [This is red],
  [and so is this]
)

- But this is not
```

However, if you set the marker to an array, you can't filter out list items that use a specific marker.

**Reply by xietuhuo** [reply]
Three level list is necessary for me. So I customize my favorite marker with array set . I see. TKS.
